src/hooks/core/chat/useSystemMessage.ts

```diff
@@ -143,6 +149,7 @@ const useSystemMessage = () => {
         __isMessageOnlyForSystemUser(sortedMessages[0]) &&
         __isMySystemMessage(sortedMessages[0])
       ) {
+        if (withLog) console.log('checkpoint4');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same as above, consider removing `console.log` or replace it with a proper logging library.



<!-- This is an auto-generated comment by OSS CodeRabbit -->